CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ Package-specific changes not released in any SDK will be added here just before
 
 ### 🐛 Bug fixes
 
+### 💡 Others
+
+- Refactored inline Android emulator checks to use enhanced checking in `EmulatorUtilities.isRunningOnEmulator()`. ([#16177](https://github.com/expo/expo/pull/16177)) by [@kbrandwijk](https://github.com/kbrandwijk), [@keith-kurak](https://github.com/keith-kurak))
```

Comment:
You don't have to update the root changelog, just update changelogs for relevant packages. We have a script that generates the root changelog from package-specific changelogs (we run it only during the SDK release) 😉

Reply:
Apologies, thanks for letting me know! I assumed I had to update the root changelog since I was also changing Expo Go code.